src/Movies.js

```diff
@@ -0,0 +1,60 @@
+import React, { Component } from "react";
+
+export default class Movies extends Component {
+	constructor() {
+		super();
+		this.state = {
+			allMovies: [],
+			currentTitle: null,
+			display: false,
+		};
```

Comment:
```suggestion
		this.state = {
			allMovies: [],
			currentTitle: null,
		};
```
You don't actually need `display` in state. Keeping `display` in state is redundant because you can just use `currentTitle` instead. In general, it's best practice to **keep only necessary things in state**.